doc/tutorials/cloud/00_setup.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>The setup needs to move to the notebook connector and be connected to the secret store. Please create a ticket,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p><a href="https://github.com/exasol/data-science-sandbox/issues/37" target="_blank">https://github.com/exasol/data-science-sandbox/issues/37</a></p>